tensorflow/compiler/mlir/hlo/include/mlir-hlo/Dialect/mhlo/IR/lhlo_ops.td

```diff
@@ -43,14 +43,16 @@ include "mlir/Interfaces/ViewLikeInterface.td"
 include "mlir-hlo/Dialect/mhlo/IR/lhlo_dialect.td"
 include "mlir-hlo/Dialect/mhlo/IR/lhlo_ops_base.td"
 include "mlir-hlo/Dialect/mhlo/IR/lhlo_ops_structs.td"
+include "mlir-hlo/Dialect/mhlo/IR/lhlo_structured_interface.td"
```

Comment:
Seems like we're missing this file?

Reply:
Added.
Should be no other missing files now.

Reply:
The ci failure seems to be non-related. 
Could you help to have a look?